target/linux/mvebu/config-4.14

```diff
@@ -407,6 +409,9 @@ CONFIG_PM_OPP=y
 CONFIG_PRINTK_SAFE_LOG_BUF_SHIFT=11
 CONFIG_PWM=y
 CONFIG_PWM_SYSFS=y
+CONFIG_POWER_RESET=y
+# CONFIG_POWER_RESET_QNAP is not set
+CONFIG_POWER_RESET_WTMCU=y
```

Comment:
I would really like this module to be added as kmod package, but if it's not possible then it's ok.

Reply:
I'll look into

Reply:
What if poweroff or reboot occurs before the module is loaded? Board will just be stuck, with the only way out via unplugging the power supply.

Reply:
You know what, there is very similar driver in kernel `/drivers/power/reset/qnap-poweroff.c`. I'll try to add mine as well (or add a patch to qnap) into the upstream alongside with the DTS. Guess it'd be the best option.

Reply:
That would be nice.